moveit_planners/ompl/ompl_interface/src/ompl_interface.cpp

```diff
@@ -71,10 +88,41 @@ OMPLInterface::OMPLInterface(const moveit::core::RobotModelConstPtr& robot_model
   RCLCPP_DEBUG(LOGGER, "Initializing OMPL interface using specified configuration");
   setPlannerConfigurations(pconfig);
   loadConstraintSamplers();
+
+  store_planner_data_service_ = node_->create_service<std_srvs::srv::Trigger>(
+      "store_planner_data",
+      [this](const std::shared_ptr<std_srvs::srv::Trigger::Request>& /* unused */,
+             const std::shared_ptr<std_srvs::srv::Trigger::Response>& response) -> void {
+        bool success = context_manager_.storePlannerData();
```

Comment:
this should call `OMPLInterface::storePlannerData()` instead of copying the function body

Reply:
Previous commit just does that by calling std::bind, but it fails clang-tidy. Any way around that?